actionview/lib/action_view/helpers/tag_helper.rb

```diff
@@ -420,6 +420,14 @@ def method_missing(called, *args, escape: true, **options, &block)
       #   # A void element:
       #   tag.br  # => <br>
       #
+      # Note that when using the block form you'll need to wrap options in
+      # parentheses.
```

Comment:
We try to [avoid](https://edgeguides.rubyonrails.org/api_documentation_guidelines.html#pronouns) using "you" in the API docs.
Maybe something like:
```suggestion
      # Note that when using the block form options should be wrapped
      # in parentheses.
```

Reply:
Done!